src/app/lists/ListDataview.tsx

```diff
@@ -74,7 +101,26 @@ const ListsDataView: React.FC<Props> = (props) => {
         throw new Error("No data found");
     }
 
-    const toggleableHeaders = headerKeysAndLabels.map(([key]) => key);
+    const rows = props.data.map((row) => {
+        const newRow: ListRow = {
+            itemName: row.item_name,
+            "1": { quantity: row["1_quantity"], notes: row["1_notes"] },
+            "2": { quantity: row["2_quantity"], notes: row["2_notes"] },
+            "3": { quantity: row["3_quantity"], notes: row["3_notes"] },
+            "4": { quantity: row["4_quantity"], notes: row["4_notes"] },
+            "5": { quantity: row["5_quantity"], notes: row["5_notes"] },
+            "6": { quantity: row["6_quantity"], notes: row["6_notes"] },
+            "7": { quantity: row["7_quantity"], notes: row["7_notes"] },
+            "8": { quantity: row["8_quantity"], notes: row["8_notes"] },
+            "9": { quantity: row["9_quantity"], notes: row["9_notes"] },
+            "10": { quantity: row["10_quantity"], notes: row["10_notes"] },
+        };
+        return newRow;
+    });
+
+    const toggleableHeaders = headerKeysAndLabels
+        .map(([key]) => key)
+        .filter((key) => key !== "itemName");
```

Comment:
do you think it's worth replacing the `.map.filter` with just `.filter(([key]) => key !== "itemName")`?